### DISCUSSION TITLE
Model retraining

### DISCUSSION THREAD

**Discussion by OmbrettaBistoni** [google/lightweight_mmm#241] [Q&A]
Hi,
I am writing because I find this model extremely interesting. 
I was wondering if you have considered the idea of continuously train this model. For example if I have daily data, should I train it every day? How many data should I consider? 
If, for example, I firstly trained the model on data of a channel that is no longer available, how should I behave about the results? And what about a new channel introduced? How long should I wait until I can consider it, too?

Thanks a lot

**Comment by statm3n** [comment]
Thanks for asking those questions. I would say some of your questions are research directions and would require domain business knowledge to answer. LightweightMMM provides a solution to model marketing activities towards business objectives. It would be dependent on the user in applying it to real business problem.
